common/djangoapps/third_party_auth/provider.py

```diff
@@ -18,17 +19,17 @@ def _enabled_providers(cls):
         """ Helper method to iterate over all providers """
```

Comment:
Update docstring to indicate that this returns an iterator over all providers for the current site.